netlify/functions/render/index.ts

```diff
@@ -0,0 +1,15 @@
+import { builder, type Handler } from '@netlify/functions';
```

Comment:
I forgot, but we still need SSR rendering due to the server components. I'm going to play around and see if I can get this to do the render for the editor.